ibm_mq/README.md

```diff
@@ -179,6 +179,8 @@ There are three service checks:
 `ibm_mq.can_connect`: checks if we can connect to IBM MQ
 `ibm_mq.queue_manager`: checks if the Queue Manager is working
 `ibm_mq.queue`: checks if the queue exists
+`ibm_mq.channel`: checks if can connect to channel
+`ibm_mq.channel.status`: checks if channel is running based on it's status
 
```

Comment:
This section should be fixed to follow the standard.
- Remove the line `There are three service checks:`
- List each check like the following examples. Note:
    - There is a **double space** after the `:`. This tells markdown to create a line down or `<br>`.
    - Separate each check with an additional line down.
    - The descriptions here and in the JSON should match.

```
**ibm_mq.can_connect**:  
Returns `CRITICAL` if the Agent cannot connect to the MQ server, otherwise returns `OK`.

**ibm_mq.queue_manager**:  
Returns `CRITICAL` if the Agent cannot retrieve stats from the queue manager, otherwise returns `OK`.
```

Reply:
Sure. Thx for the suggested. The section has been updated.

Reply:
Are those standards written somewhere? or a real integration that we can use as reference?

I'm a bit concerned about the `"double space" after the :`, usually it's a bad practice in programming to have trailing spaces and many software engineers have automatic trailing spaces deletion activate on purpose.

I updated with `<br/>` instead, let's me know if that ok for you. It's not very elegant to have `<br>` but IMHO that's more readable and less prone to error than invisible double spaces.

Reply:
Sounds good